Patches/Arasaka Corporation [1.3]/Cyberweeb_Apperal.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+
+  <Operation Class="PatchOperationFindMod">
+    <mods>
+      <li>Arasaka Corporation [1.3]</li>
+    </mods>
+
+    <match Class="PatchOperationSequence">
+      <operations>
+
+
+      <!-- === Apparel_Turtleneck_AC === -->
+
+      <li Class="PatchOperationAdd">
+        <xpath>Defs/ThingDef[defName="Apparel_Turtleneck_AC" or defName="Apparel_Workwear_AC" or defName="Apparel_Workwear_W_AC"]/statBases</xpath>
+        <value>
+          <Bulk>4</Bulk>
+          <WornBulk>0.5</WornBulk>

```

Comment:
WornBulk seems sort of low for this apparel. The turtle neck is a long-sleeved shirt, and the workwear is an entire jumpsuit, so they should probably have 1 - 2 WornBulk, like most other apparel. Any special reason to have it this low?

Reply:
> WornBulk seems sort of low for this apparel. The turtle neck is a long-sleeved shirt, and the workwear is an entire jumpsuit, so they should probably have 1 - 2 WornBulk, like most other apparel. Any special reason to have it this low?

AFAIK, most skin layer apparel like shirt and pants have zero bulk.